samples/nrf9160/lwm2m_client/src/main.c

```diff
@@ -470,15 +471,9 @@ void main(void)
 	lwm2m_verify_modem_fw_update();
 #endif
 #if !defined(CONFIG_NRF_MODEM_LIB_SYS_INIT)
-	ret = at_cmd_init();
-	if (ret != 0) {
-		LOG_ERR("at_cmd_init failed: %d\n", ret);
-		return;
-	}
-
-	ret = at_notif_init();
+	ret = nrf_modem_at_init();
```

Comment:
Do we really need to do this? I haven't seen any other component initializing `nrf_modem_at`.
Isn't `nrf_modem` library handling this as part of its initialization whether it's initialized as part of sys init or init triggered manually to `nrf_modem`?
@lemrey, please comment.

Reply:
@trantanen True. I didn't see that above there is a similar macro section that manually starts libmodem when `CONFIG_NRF_MODEM_LIB_SYS_INIT` is false. I'll update accordingly.